docs/application/dotnet/guides/nui/flex-layout.md

```diff
@@ -110,8 +110,78 @@ flexLayout.WrapType = FlexLayout.FlexWrapType.NoWrap;
 layoutView.Layout = flexLayout;
 ```
 
+ The following properties affect the specified child view areas:
+- `FlexAlignmentSelf`
+-  `FlexPositionType`
+-  `FlexAspectRatio`
+-  `FlexBasis`
+-  `FlexShrink`
+-  `FlexGrow`
 
-## Related Information
+> [!NOTE]
+> The `FlexAlignmentSelf`, `FlexPositionType`, `FlexAspectRatio`, `FlexBasis`, `FlexShrink`, and `FlexGrow` properties are added in Tizen 6.0.
+
+**Table: Properties and descriptions**
+
+| Property               | Type            | Description  |
+| -----------------------| --------------- | ------------ |
+| `FlexAlignmentSelf`    | AlignmentType   | The alignment of the item along the cross axis when free space is available. |
+| `FlexPositionType`     | PositionType    | The position type defines how it is positioned within its parent. |
+| `FlexAspectRatio`      | float           | The ratio between the width and the height of an item. |
+| `FlexBasis`            | float           | The item's default size before any flex grow and flex shrink calculations are performed. |
+| `FlexShrink`           | float           | Specifies how to shrink children along the main axis in the case that the total size of the children overflows the size of the flex layout on the main axis. |
+| `FlexGrow`             | float           | Specifies how space within a flex layout is distributed among its children along the main axis. |
+
+## FlexAlignmentSelf property
+
+The `FlexAlignmentSelf` property has the same options and effect as `Alignment` property, but instead of affecting the children within a flex layout, this property is applied to a single child to change its alignment within its parent. The align-self overrides any option set by the parent with align-items.
```

Comment:
```suggestion
The `FlexAlignmentSelf` property has the same options and effect as the `Alignment` property, but instead of affecting the children within a flex layout, this property is applied to a single child to change its alignment within its parent. The align-self overrides any option set by the parent with align-items.
```

If align-self,  align-items are properties we need to keep them in code style.

Reply:
Done.